examples/zero_copy/README.md

```diff
@@ -0,0 +1,29 @@
+# A chatter example demonstrating shared memory backend for zero copy transport
+
+ROS2 supports using shared memory to speed up transport of messages. First, run shared memory manager:
+
+```sh
+bazel run @iceoryx//:shared_memory_manager
+```
+
+Then, for each node started, specify the CycloneDDS config file:
+
+```sh
+export CYCLONEDDS_URI=file://<path to rules_ros2>/examples/zero_copy/configs/cyclonedds.xml
+bazel run //examples/chatter:talker
```

Comment:
Off the top of my head, I would suggest interested people run this in opt mode, as e.g. `--config=opt`. WDYT?

In principle, as iceoryx and cyclonedds are compiled in release mode, that should already help.